src/views/HomeView.js

```diff
@@ -188,8 +188,11 @@ class HomeView extends AbstractView {
 
       for (const area of Helper.areas) {
         if (!Helper.strategyOptions.areas[area.area_id]?.hidden) {
+          let options = Helper.strategyOptions.areas[area.area_id ?? "undisclosed"];
```

Comment:
Import the correct module at line #186, depending on the type of `Helper.strategyOptions.areas["*"]`.

Reply:
Done

Reply:
Actually, you didn't.

Instead of importing the correct module at line #186, you import both modules at line #3 and #4.
I requested something like:

```js
import("../cards/" + (Helper.strategyOptions.areas["_"]?.type ?? "AreaCard")).then(areaModule => {
```

You keep the module name `HaAreaCard`, but rename the class from `HaAreaCard` to `AreaCard`.
This way only 1 module will be imported (instead of 2) and we avoid an extra check for which class to instantiate.

Reply:
Ok! I'll change it